draft-ietf-taps-interface.md

```diff
@@ -1278,6 +1281,19 @@ Size for the Connection, the Send will fail with a SendError event ({{send-error
 example, it is invalid to send a Message over a UDP connection that is larger than
 the available datagram sending size.
 
+## Sending Replies {#send-replies}
+
+When a message is sent in responses to a message received, the application
+may use the Message Context of the received Message to construct a Message Context for the reply.
+
+~~~
+replyMessgeContext := reqestMessageContext.reply()
+~~~
+
+By using the ```replyMessgeContext```, the transport system is informed that
```

Comment:
just a nit: here, and in the "code" line above, s/replyMessgeContext/replyMessageContext